src/test/java/core/basesyntax/strategy/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.dao.FruitsDao;
+import core.basesyntax.dao.FruitsDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.FruitHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static FruitTransaction fruitTransaction;
+    private static OperationStrategy operationStrategy;
+    private static FruitsDao fruitsDao;
+    private static Map<FruitTransaction.Operation, FruitHandler> fruitHandlerMap;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitsDao = new FruitsDaoImpl();
+        fruitHandlerMap = new HashMap<>();
+        fruitHandlerMap.put(
+                FruitTransaction.Operation.BALANCE, new BalanceOperationHandler(fruitsDao));
+        operationStrategy = new OperationStrategyImpl(fruitHandlerMap);
```

Comment:
Do we need to make everything so complicated? Why don't you just use BalanceHandler here?

Reply:
Check all your handlers tests and improve them